starter-project-backend-g31/src/controllers/rating.ts

```diff
@@ -1,4 +1,6 @@
 import { Rating } from "../models/rating";
+import {Article} from '../models/article';
+import {addRatingToArticle, updateRatingForArticle} from "./article"
 
 export const getAllRatings = (req: any, res: any) => {
```

Comment:
we may not need this part alone, getAllRating for  a given aritcle makes sense but without article it does not have functionality.

Reply:
Yeah I removed the import for Article. but the second two are for interlinking purposes.